lib/Migration/Version000203Date20200322200200.php

```diff
@@ -67,31 +68,39 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 
 	/**
 	 * @param IOutput $output
-	 * @param \Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
 	 * @param array $options
+	 *
 	 * @since 13.0.0
 	 */
-	public function postSchemaChange(IOutput $output, \Closure $schemaClosure, array $options) {
-		$qb = $this->connection->getQueryBuilder();
-
-		$cursor = $qb->select(array('id', 'aaguid'))
-		   ->from('twofactor_webauthn_registrations')
-		   ->execute();
-
-		while ($row = $cursor->fetch()) {
-			$updater = $this->connection->getQueryBuilder();
-			$updater->update('twofactor_webauthn_registrations')
-				->set('aaguid_transform', $updater->createNamedParameter($this->getUuidString($output, $row)))
-				->where('id = :id')
-				->setParameter('id', $row['id'])
+	public function postSchemaChange(IOutput $output, Closure $schemaClosure, array $options) {
+		$this->connection->beginTransaction();
+		try {
+			$selectQb = $this->connection->getQueryBuilder();
+			$result = $selectQb->select('id', 'aaguid')
+				->from('twofactor_webauthn_registrations')
 				->execute();
+			$updateQb = $this->connection->getQueryBuilder();
+			$updateQb->update('twofactor_webauthn_registrations')
+				->set('aaguid_transform', $updateQb->createParameter('aaguid_transform'))
+				->where($updateQb->expr()->eq('id', $updateQb->createParameter('id')));
```

Comment:
No need to "try" this? same with the selectQb before it could be outside of the try-catch
Same in the other migrations